### ISSUE TITLE
Cannot see fuzzer's log at local testing?

### ISSUE THREAD

**Issue by sky-bro** [google/clusterfuzz#1788]
I'm running clusterfuzz locally (using v2.0.1), and now I can create a job at `localhost:9000/jobs` page and submit it at `localhost:9000/fuzzers` page.
Everything is working just fine except at the fuzzers page, I cannot view a fuzzer's logs: `http://localhost:9000/gcs-redirect?path=/test-fuzzer-logs-bucket/libFuzzer`, it redirects me to `https://console.cloud.google.com/storage/browser/test-fuzzer-logs-bucket/libFuzzer`, and the `last result` always gives me `Not run yet`.
I though locally the `fuzzer's logs link` should bring me to some locally emulated google cloud storage page? Or maybe this is just not supported for local testing?

**Comment by inferno-chromium** [comment]
yes, that link wont work in local testing, you can see those files in <clusterfuzz_root_dir>/local/storage. we can try fixing this, but not a priority atm since we focus on production workflow. patches welcome, should be a line or two change in gcs_redirector.py. if local, redirect to file:// path to local/storage.
